.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,56 @@
+<!--
+
+	Note that we use GitHub issues for bugs and (uncontroversial) feature
+    requests (see below for details).
+
+    Please do *NOT* ask usage questions in GitHub issues.  Usage questions make
+	most sense on the users mailing list, where more people are available to
+	potentially respond to your question, and the whole community can benefit
+	from the answers provided (perhaps your question has already been answered,
+	search the archive to find out):
+	https://groups.google.com/forum/#!forum/prometheus-users
+
+	While a GitHub issue is fine to track progress on an uncontroversial
+	feature request, many feature requests touch the best practices and
+	concepts of Prometheus as a whole and need to be discussed with the wider
+	developer community first. This is in particular true for a request to
+	reconsider a prior rejection of a feature request or any of the declared
+	non-goals (see README.md). Those overarching discussions happen on the
+	developer mailing list (GitHub issues, in particular closed ones, are not
+	tracked by the wider developer community and thus inadequate):
+	https://groups.google.com/forum/#!forum/prometheus-developers
+	
+    You can find more information at: https://prometheus.io/community/
+
+-->
+
+## Feature request
+**Use case. Why is this important?**
+
+*Nice to have' is not a good use case :)*
```

Comment:
There's a single quote / apostrophe after "have" that doesn't have a corresponding opening quote?